test/functional/specs/C2587.js

```diff
@@ -26,16 +24,13 @@ test.meta({
 });
 
 const bogusCommand = ClientFunction(() => {
-  window.alloy("bogusCommand");
+  return window.alloy("bogusCommand").then(() => {}, error => error.message);
 });
 
-test.before(async () => {
+test("Test C2587: Throw error when executing command that doesn't exist", async t => {
   await configureAlloyInstance("alloy", config);
-  await bogusCommand();
-})(
-  "Test C2587: Throw error when executing command that doesn't exist",
-  async t => {
-    const { error } = await t.getBrowserConsoleMessages();
```

Comment:
This was testing that an error was logged to the console rather than testing that the command's promise was actually rejected.